app/waf/package.json

```diff
@@ -62,6 +62,7 @@
     "@types/uuid": "^3.4.4",
     "loopback-connector-postgresql": "^3.5.0",
     "loopback-connector-rest": "^3.4.1",
+    "loopback-connector-rest-hdr": "^3.4.4",
```

Comment:
Are we planning to use this one until loopback-connector-rest supports response header, right?

Reply:
Yeah, their response is slow... so let's maintain our own packages for a while.